### DISCUSSION TITLE
GitHub "Support" is inaccessible and badly designed

### DISCUSSION THREAD

**Discussion by JohnAlexINL** [community/community#126070] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Just to get to a place where I can even begin talking about any problem I'm having, I needed to:

1. Log in and find `GitHub Support` from the drop-down menu. If I had any problems logging in, I guess I'm just boned
2. From there, scroll nearly to the bottom of the page to find the relevant subject
3. On that page, scroll past all of the "News" and other _crap_ that is entirely irrelevant to _getting support_, and past the "Still Need Help" section because it doesn't actually have help options (just more links back to the blog and FAQs) to the "Contact Us" button which takes me to `https://support.github.com/request`
4. From here, it gives me a "Virtual Assistant" chat that will not and cannot help with the issue because the issue is not one of the fifteen things the people who made this thing expected and pre-crafted prompts for, so it gets stuck in a loop of just asking the same irrelevant question over and over again. There is, however, a link at the top-right of the page to view "My Tickets",
5. From the tickets page, `https://support.github.com/tickets/personal`, I can click "New Ticket", which .... oh, wait, that just brings me back to step 4. Isn't that helpful? No, see, in that menu where you pick what the type of problem you're having, if you actually want to submit a ticket you need to click "Provide Feedback". The other options just give you the _entirely useless_ "Virtual Assistant" (or would definitely be the wrong place for the issue I was having). 
6. From here (`https://github.com/orgs/community/discussions`) you can click "New Discussion" and it brings you to another ****ing menu asking you to select what kind of problem you're having, and most of the options from earlier are also gone, so there isn't even an option for the problem I'm having. I guess you just choose "General" when you're having security issues? Like this is absurd
7. Once you click "Get Started" on the subject that fits whatever problem you're stuck on, you are *finally* brought here, where it asks you to describe the problem. It feels intentionally badly designed, like GitHub doesn't *want* support tickets or something, and I for the life of me cannot understand how this got past any sort of QA

**Comment by JohnAlexINL** [comment]
I haven't even been able to submit a ticket for the original problem because I either end up in Discussions, or in the endless loop with the "Virtual Assistant". I'm here because 2FA is borked (again) and I wanted to file a bug report, but the only option it actually gives you is to reset your account. I don't need to recover the account tho, I have other devices that are still logged in, but I wanted to report the bug and stuff

**Reply by davevad93** [reply, marked as answer]
Hi @JohnAlexCO ,

I don't know if that's what you're looking for, but try with raising a ticket [here](https://support.github.com/contact/account) and select one between `Two-factor authentication recovery` or `Sign in, verification or other account questions` if you're experiencing a problem with 2FA.

**Reply by Eltalvazquez** [reply]
alguien puede ayudarme?no soy desarrollador, estoy bajo un ataque ddos y mi telefono personal esta corriendo un softweare de prueba de algun repositirio, por favor ayuda!

**Reply by Eltalvazquez** [reply]
no entiendo nada de programacion, entre sin saber a un repositorio y los ajustes de un softweare de prueba estan corriendo en mi telefono personal y otros programadores y bots realizan ajustes que me impiden usarlo con seguridad y estoy bajo un ataque de ddos y tengo aplicaciones clonadas que tienen permisos para controlar el dispositivo. no se que hecer. explicame por favor si puedes ayudarme o como?

**Reply by grandmas-favorite** [reply]
Hi @Eltalvazquez, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting.

**Reply by Eltalvazquez** [reply]
(I don't understand anything about programming, I entered a repository without knowing and the settings of a test software are running on my personal phone and other programmers and bots make settings that make me They prevent me from using my device and I have cloned applications with permissions that control the phone...please help!

**Comment by DengxinH** [comment]
easy, they dont want you to get any support.

**Comment by MikeSchoenfelder-MIPS** [comment]
I completely agree.  Why is it taking more than 30 minutes to search for how to file a billing ticket that I know can't be answered by the community?  I asked CoPilot how to file a support ticket and it said that question was outside of its training!  That does not give me confidence that CoPilot will be helpful with anything.

**Reply by JohnAlexINL** [reply]
you ask for AI-generated slop, you get AI-generated slop. I do not understand how this has anything to do with the issue I filed last year
